applications/GeoMechanicsApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -175,7 +175,7 @@ void AddCustomProcessesToPython(pybind11::module& m)
 
     py::class_<ApplyCPhiReductionProcess, ApplyCPhiReductionProcess::Pointer, Process>(
         m, "ApplyCPhiReductionProcess")
-        .def(py::init<ModelPart&, const Parameters>());
+        .def(py::init<Model&, const Parameters>());
```

Comment:
There was a mistake here before: the second parameter should be `const Parameters&`, since that is the type declared by the constructor:

```suggestion
        .def(py::init<Model&, const Parameters&>());
```

Reply:
done